packages/server/tests/helpers/prerequisite.ts

```diff
@@ -11,19 +11,16 @@ const RELAY_URL = process.env.E2E_RELAY_HOST || LOCAL_RELAY_URL;
 
 (function () {
   if (USE_LOCAL_NODE) {
-    process.env['NETWORK_NODE_IMAGE_TAG'] = '0.45.0-alpha.0';
-    process.env['HAVEGED_IMAGE_TAG'] = '0.45.0-alpha.0';
-    process.env['MIRROR_IMAGE_TAG'] = '0.92.0';
+    const networkNodeImage = '0.47.0-alpha.4';
```

Comment:
Q: why no longer process env variables.
Doing it as is means we can't override without code changes

Reply:
Removed them, local node will be up to date with the latest images always, so no need here to do anything.